core/src/accounts_hash_verifier.rs

```diff
@@ -39,7 +41,7 @@ impl AccountsHashVerifier {
         cluster_info: &Arc<ClusterInfo>,
         trusted_validators: Option<HashSet<Pubkey>>,
         halt_on_trusted_validators_accounts_hash_mismatch: bool,
-        fault_injection_rate_slots: u64,
+        fault_injection_rate_slots: Slot,
```

Comment:
I feel better to leave this u64 -- it is the rate of slots, right -- fundamentally a number?

Reply:
Yeah, I agree. Ideally I'd like this to be some sort of difference type too.

Since Rust doesn't let you do implicit conversions within arithmetic, and the actual slot is compared to the interval, one of the two needs to be cast to the other. I think some of the older code uses `u64` instead of `Slot` likely before the `Slot` alias existed. It seems other parts of the code uses `Slot` for the various rates as well, so I was trying to move in that direction here. Like here:

https://github.com/solana-labs/solana/blob/f59a55be171e64aa7deb021cad756ca3c69968cc/runtime/src/snapshot_config.rs#L8-L13

I'm fine using any type here, but I would like to understand what the overall best practice is/should be for this codebase, so I don't further bifurcate it :)

@carllin What do you think about this?

Reply:
It's counted in slots, so I see no problem with using that as the unit of measurement